### PR TITLE
Modifying the ybvoyager_visualizer_metadata table through ALTER DDL

### PR DESCRIPTION
**Pull request by sanyamsinghal**
- so that visualizer info of previous voyager version and current voyager version can co-exist

### CODE REVIEW

File: yb-voyager/src/cp/yugabyted/yugabyted.go [hunk 1]
```diff
@@ -447,12 +449,71 @@ func (cp *YugabyteD) createYugabytedMetadataTable() error {
...
+	err := cp.executeCmdOnTarget(cmd)
+	if err != nil {
+		return err
+	}
+
+	alterTableCmds := []string{
+		fmt.Sprintf(`
+        DO $$
+        BEGIN
+            IF NOT EXISTS (SELECT 1 FROM information_schema.columns
```
Comment (priyanshi-yb) [inline yb-voyager/src/cp/yugabyted/yugabyted.go:463]: @sanyamsinghal I believe there is `ALTER TABLE .. ADD COLUMN IF NOT EXISTS ...` clause as well. you can maybe directly use that. 
https://www.postgresql.org/docs/current/sql-altertable.html#:~:text=ADD%20%5B%20COLUMN%20%5D%20%5B%20IF%20NOT%20EXISTS%20%5D%20column_name%20data_type%20%5B%20COLLATE%20collation%20%5D%20%5B%20column_constraint%20%5B%20...%20%5D%20%5D

Comment (sanyamsinghal) [reply yb-voyager/src/cp/yugabyted/yugabyted.go:463]: thanks @priyanshi-yb 
As per the documentation, it is from PG 9.6 onwards.
Not mentioned in version before that, but i am not sure if its just not documented or not supported also.

Comment (sanyamsinghal) [reply yb-voyager/src/cp/yugabyted/yugabyted.go:463]: Will have to check, since we certify PG 9.x-11.x
So that means PG >= 9

Comment (sanyamsinghal) [reply yb-voyager/src/cp/yugabyted/yugabyted.go:463]: oh ok, i got a bit confused. 
Since YB uses PG11 so should not be a problem.

Comment (sanyamsinghal) [reply yb-voyager/src/cp/yugabyted/yugabyted.go]: done

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3422/

**Review by makalaaneesh — APPROVED** [review]
LGTM. Pls manually test voyager upgrade (1.7.2->1.8) and rollback(1.8->1.7.2) scenarios with the same yugabyted instance

**Comment by sanyamsinghal** [comment]
> Pls manually test voyager upgrade (1.7.2->1.8) and rollback(1.8->1.7.2) scenarios with the same yugabyted instance

@makalaaneesh Upgrade from `1.7.2` -> `1.8` would fail for yugabyted, as we know already.
I think you to upgrade from 1.7.2 to this branch and rollback to 1.7.2 ?

**Pull request merged**